### DISCUSSION TITLE
Lesson 7: I can not install chainlink dependencie

### DISCUSSION THREAD

**Discussion by ely67** [Cyfrin/foundry-full-course-cu#1249] [General]
I'm running 'forge install smartcontractkit/chainlink-brownie-contracts@0.8.0 --no-commit' to install Chainlink dependencies, but I encountered the following error: 

'error: 2871 bytes of body are still expected MiB | 300.00 KiB/s
fetch-pack: unexpected disconnect while reading sideband packet
fatal: early EOF
fatal: fetch-pack: invalid index-pack output
fatal: clone of 'https://github.com/smartcontractkit/chainlink-brownie-contracts' into submodule path '/home/elyas6126/foundry-f23/fund-me/lib/chainlink-brownie-contracts' failed
Error: 
git submodule exited with code 128'.

please help me

**Comment by Ehsan-The-Coder** [comment]
### **i was facing the same issue but resolve with one simple way** 

```
forge install smartcontractkit/chainlink-brownie-contracts@0.8.0 --no-commit
Installing chainlink-brownie-contracts in D:/BlockChain/Project/Showcase/FoundryFullCourse/FundMe/lib\chainlink-brownie-contracts (url: Some("https://github.com/smartcontractkit/chainlink-brownie-contracts"), tag: Some("0.8.0"))
Cloning into 'D:/BlockChain/Project/Showcase/FoundryFullCourse/FundMe/lib/chainlink-brownie-contracts'...
remote: Enumerating objects: 32923, done.
remote: Counting objects: 100% (2524/2524), done.
remote: Compressing objects: 100% (1050/1050), done.
error: RPC failed; curl 92 HTTP/2 stream 5 was not closed cleanly: CANCEL (err 8)
error: 3081 bytes of body are still expected
fetch-pack: unexpected disconnect while reading sideband packet
fatal: early EOF
fatal: fetch-pack: invalid index-pack output
fatal: clone of 'https://github.com/smartcontractkit/chainlink-brownie-contracts' into submodule path 'D:/BlockChain/Project/Showcase/FoundryFullCourse/FundMe/lib/chainlink-brownie-contracts' failed
Error: 
git submodule exited with code 128
```



**Issue:** my issue was that i initiated git in the FoundryFullCourse inside this i have all lesson folders

**Solution:** initiate the git with `git init` in the working directory and then try to install with 
``` 
forge install smartcontractkit/chainlink-brownie-contracts@0.6.1 --no-commit
```

**Reply by qingrun-tan** [reply]
forge install smartcontractkit/chainlink@v2.7.2 --no-commit

**Reply by qingrun-tan** [reply]
you address is error

**Reply by Ehsan-The-Coder** [reply]
try this 
Solution: 
1:-  initiate the git with `git init` in the working directory and then try to install with
2:- `forge install smartcontractkit/chainlink-brownie-contracts@0.6.1 --no-commit`

**Comment by ely67** [comment]
thank you
